python/sdist/amici/importers/sbml/__init__.py

```diff
@@ -1904,6 +1906,23 @@ def _process_events(self) -> None:
                 "priority": self._sympify(event.getPriority()),
             }
 
+            if self.jax:
+                if not initial_value:
```

Comment:
bit more nuanced than than, let's just not support events with initial values for now.